### DISCUSSION TITLE
I have a problem with unit conversion because molecular weight is not available in version 10

### DISCUSSION THREAD

**Discussion by sajabzughoul** [Open-Systems-Pharmacology/Forum#1037] [Questions & Problems] [answered]
![Capture](https://user-images.githubusercontent.com/45569077/144076614-809da740-5941-443d-8355-9a5e7339efc9.JPG)

**Comment by StephanSchaller** [comment]
To understand the issue and for us to help, you will need to provide some background on what you did.

**Comment by sajabzughoul** [comment]
Dealing with PK-Sim version 10 for population templates and doing simulations I need with my project. I can't export the data (conc-time) profile of simulations that have already done on pk-sim version 9 because of molecular weight not available for that compound in version 10

**Comment by msevestre** [comment]
Where you able to export in v9 with in umol/l? 
In v10, try to change the unit to mg/l. Is it PKSim or MoBi

**Reply by sajabzughoul** [reply]
<img width="960" alt="Pop" src="https://user-images.githubusercontent.com/45569077/144276278-78a90395-ba58-400c-99cb-313bf4681f67.png">
<img width="960" alt="pk- shot" src="https://user-images.githubusercontent.com/45569077/144276319-0f2e28f0-4242-4798-b2be-03c8f9fbcbd9.png">
PK-Sim 
the unit was changed, but I cannot export the data  
yes, no problem in version 9 I can change the units and export the data easily whereas in pk-sim version10 I cannot except for simulations I have already done on version 10

**Comment by msevestre** [comment]
Can you attach the project (or send it to me michael@design2code.ca) so that I can check it out?

**Reply by sajabzughoul** [reply]
Sure, I will send it as link on Google Drive

**Comment by msevestre** [comment, marked as answer]
So there reason is that there is no Molecular weight associated with your obs data
`150 and 300 single dose observed.150 single dose obs`
![image](https://user-images.githubusercontent.com/1041237/144304364-01cff009-f4ba-4e0b-b0ea-5603baa73f60.png)

I am not sure how you imported this data, but this is missing. Did you associate it with a compound defined in PKSim? The imported has changed between v9 and v10 and linking to a molecule is NOT mandatory anymore.  

see another compound with a MW associated
![image](https://user-images.githubusercontent.com/1041237/144304542-6f060430-1ab5-4d50-b4b9-39e182c26e27.png)

So I would suggest to reimport the data linking it to a molecule using
![image](https://user-images.githubusercontent.com/1041237/144304614-0aee2374-07e2-43fa-859d-7e2ff19c3201.png)


and you should be good to go

**Reply by msevestre** [reply]
BTW you can export in mg/l (as long as you do not use mol units, this works). I just tried with your file, no problem

**Comment by sajabzughoul** [comment]
But still i am facing the same problem :(
can I send you the observed data trying to fix that and send me a copy?

**Reply by msevestre** [reply]
Sure. But try to follow the documentation first. You need to assign a compound to the observed data you are importing

**Reply by sajabzughoul** [reply]
Thank you, I did it 
I want to ask about a Populations for CKD stages in PK-Sim version 10, did they account for CYP3A4 level variability published in literature? this paper includes the decrease in expression ''Modeling and predicting drug 
pharmacokinetics in patients 
with renal impairment
Expert Rev. Clin. Pharmacol. 4(2), 261–274 (2011)''
it seems that they did not account for that because the concentration in all populations was 4.3 micromole/l, and I cannot edit that even in simulation.

**Comment by msevestre** [comment]
Please create a new issue if you have another question. Thanks.
